.gitignore

```diff
@@ -102,3 +102,9 @@ Release/
 # *.ini
 log.txt
 log.txt
+
+#CMake
+/build/
+CMakeCache.txt
+*.cmake
```

Comment:
I wouldn't add this to the gitignore.

Reply:
Line 109 only.

Reply:
The *.cmake line? On what basis, it doesn't seem to affect any files that should be in git?

Reply:
Not at this time, but it would exclude future CMake modules. The rest is fine.